ipmitool/debian/changelog

```diff
@@ -1,3 +1,8 @@
+ipmitool-xcat (1.8.18) unstable; urgency=low
+  * Upgrade to 1.8.18 version
+
+ -- Victor Hu <vhu@us.ibm.com> Tue, May 30, 2017 14:41:01 -0400
```

Comment:
The time format is wrong which cause the error while running make_deb.sh script.
`-- Victor Hu <vhu@us.ibm.com> Tue, May 30, 2017 14:41:01 -0400`
-->
`-- Victor Hu <vhu@us.ibm.com> Tue, 30 May 2017 14:41:01 -0400`